cmd/kops/create_cluster.go

```diff
@@ -1203,3 +1230,37 @@ func setOverrides(overrides []string, cluster *api.Cluster, instanceGroups []*ap
 	}
 	return nil
 }
+
+func getZoneToSubnetProviderID(VPCID string, region string, subnetIDs []string) (map[string]string, error) {
+	res := make(map[string]string)
+	if VPCID == "" {
+		return res, fmt.Errorf("must specify vpc when specifying subnets")
```

Comment:
It is super-tempting to try to make VPC optional (because a subnet ID is unique, and implies a VPC) - but we can do that later :-)